src/scripts/updateVideoDateTime.ts

```diff
@@ -0,0 +1,35 @@
+import { readFileSync } from 'fs';
+import db from '../db/index';
+import { exit } from 'process';
+
+export function readJson(path: string) {
+  const data = JSON.parse(readFileSync(path, 'utf-8'));
+  return data;
+}
+
+async function updateVideoDateTime(jsonPath: string) {
+  const data = readJson(jsonPath);
+  try {
+    await db.$transaction(async (tx) => {
+      for (const { id, date, timestamp } of data) {
+        await tx.videoMetadata.update({
+          where: {
+            contentId: id,
+          },
+          data: {
+            timestamp: timestamp as number,
```

Comment:
`timestamp` will be undefined here since it does not exist in the JSON data.

Reply:
I will make changes to this.